freedomofpress/securedrop-client
# Add ReplyDownloadJob

# Description

Resolves #408

* Reply sync (download+decrypt) now occurs whenever there is a metadata sync with the server instead of every 5 seconds checking if there are new replies.
* We now use the queue to download and decrypt replies in a job called ReplyDownloadJob
* message_sync.py has been deleted and references to reply_sync and reply_thread removed
* User-facing behavior should remain the same with a few modifications to log messages

# Test Plan

1. Write a reply in the reply box and hit "Send"
2. Click between sources to check that it persists
3. Refresh, see it persist
